integrations-and-supported-tools/keras/notebooks/Neptune_Keras.ipynb

```diff
@@ -355,7 +357,7 @@
     "\n",
     "model.compile(optimizer=optimizer, loss=\"sparse_categorical_crossentropy\", metrics=[\"accuracy\"])\n",
     "\n",
-    "neptune_cbk_2 = NeptuneCallback(run=run_2, base_namespace=\"metrics\")\n",
+    "neptune_cbk_2 = NeptuneCallback(run=run_2)\n",
```

Comment:
```suggestion
    "neptune_cbk_2 = NeptuneCallback(run=run_2) # base_namespace defaults to \"training\"\n",
```
Let's add this comment here as we are removing the parameter but mentioning it in the header